themes/digital.gov/src/scss/new/guides/_guide-menu-bar.scss

```
@@ -0,0 +1,63 @@
+@use "uswds-core" as *;
+
+.dg-guide__menu-bar {
+  @include u-border-y("1px", "gray-cool-30", "solid");
+  @include u-z("top");
+  align-items: center;
+  background-color: color("gray-10");
+  display: flex;
+  height: units(10);
+  justify-content: center;
+  position: sticky;
+  top: -1px;
+  white-space: nowrap;
+
+  &-image {
+    display: none;
+    margin: 0 units(3);
+    &.full {
```

An empty line to separate new selectors helps with readability.

```suggestion
    margin: 0 units(3);
    
    &.full {
```